addons/hr/static/tests/legacy/helpers/model_definitions_setup.js

```diff
@@ -1,10 +1,5 @@
 /** @odoo-module **/
 
-import { addFakeModel, addModelNamesToFetch } from '@bus/../tests/helpers/model_definitions_helpers';
+import {  addModelNamesToFetch } from '@bus/../tests/helpers/model_definitions_helpers';
 
 addModelNamesToFetch(['hr.employee', 'hr.employee.public', 'hr.department']);
-
-addFakeModel('m2x.avatar.employee', {
-    employee_id: { string: "Employee", type: 'many2one', relation: 'hr.employee.public' },
-    employee_ids: { string: "Employees", type: "many2many", relation: 'hr.employee.public' },
-});
```

Comment:
Is this file really needed now that you have updated all the tests?

Reply:
I think this file is still required, as the test cases for many other modules have not yet been converted to Hoot and still rely on these models.